Model/Service/SendEmail.php

```diff
@@ -52,30 +52,36 @@ public function __construct(
      */
     public function with($reportHtml)
     {
-        $this->inlineTranslation->suspend();
-        $sender = [
-            'name'  => $this->escaper->escapeHtml($this->getMailSenderClearName()),
-            'email' => $this->escaper->escapeHtml($this->getMailSenderAddress()),
-        ];
-        $transportBuilder = $this->transportBuilder->setTemplateIdentifier('security_csp_report')->setTemplateOptions(
-            [
-                'area'  => \Magento\Framework\App\Area::AREA_ADMINHTML,
-                'store' => \Magento\Store\Model\Store::DEFAULT_STORE_ID,
-            ]
-        )->setTemplateVars(
-            [
-                'body' => $reportHtml,
-            ]
-        )->setFrom($sender);
+        $recepients = [];
+        try {
+            $this->inlineTranslation->suspend();
```

Comment:
the most safe way is to move `suspend()` before `try`, so that `resume()` is not called if `suspend()` itself throws an exception